c2rust-transpile/src/translator/mod.rs

```diff
@@ -3785,12 +3785,11 @@ impl<'c> Translation<'c> {
                                 Type::Tuple(TypeTuple { elems: ref v, .. }) if v.is_empty() => ReturnType::Default,
                                 _ => ReturnType::Type(Default::default(), ret_ty),
                             };
-                            let bare_ty: (Vec<BareFnArg>, Option<Variadic>, ReturnType) = (
+                            let bare_ty = (
                                 vec![mk().bare_arg(mk().infer_ty(), None::<Box<Ident>>); args.len()],
                                 None,
                                 ret_ty
                             );
```

Comment:
The type annotation here previously served as documentation for what `None` was doing here (specifying that the `fn` is not variadic), so it would be nice to replace this type with a comment or type annotation on the `None`.

Reply:
Okay, I'll do `None::<Variadic>`.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/550/commits/d3da84c875c6ff1c2b93a17055457c090a12408a.